SupportGuidelines.md

```diff
@@ -0,0 +1,73 @@
+# How to Quickly Identify Problem Faced by Community? 
+
+## TLDR;
+1. **Get as much of factual information surrounding the problem.** 
+    - Operating System
+    - Software Version
+    - Specific Program to Run
+    - Environment Setup
+    - Intention / Motivation of the action
+    
+2. **Provide workaround.**  
+If could not identify problem through the first step, focus on suggest the workarounds to solve the immediate pain,  
+   then spend time in replicating the problem on own system (in order to start solving the problem).
```

Comment:
here.

Reply:
@YCCertifai yeah, its intention for people to just read the upper parts if they do not want to read the whole article.

often it quoted with [TLDR](https://www.google.com/search?q=TLDR%3B&rlz=1C5CHFA_enMY930MY930&oq=TLDR%3B&aqs=chrome..69i57j69i59j69i58.2359j1j4&sourceid=chrome&ie=UTF-8)

Reply:
ok will merge it